get_data_endpoints/README.md

```diff
@@ -0,0 +1,106 @@
+# Fybrik KFP component
+
+This is a component that can be used in Kubeflow Pipelines to read two input datasets (ex: test and training data), and to write the results of whatever
+processing is done on that data **in a secure and governed fashion by leveraging Fybrik**
+
+It is assumed that Fybrik is installed together with the chosen Data Catalog and Data Governance engine, and that:
+
+* training and testing datasets have been registered in the data catalog
+* governance policies have been defined in the data governance engine
+
+## Prerequisits
+
+* [Install kubeflow pipelines](https://www.kubeflow.org/docs/components/pipelines/installation/overview/#kubeflow-pipelines-standalone)
+  * To install on kind, see [these instructions](https://github.com/machine-learning-exchange/mlx/blob/main/docs/install-mlx-on-kind.md#install-kubeflow-pipelines-for-reference-only-optional)
+* [Install Fybrik](https://fybrik.io/v1.0/get-started/quickstart/)
+* [Deploy Datashim](https://github.com/datashim-io/datashim)
+
+This component is compatible with Fybrik v1.0.
+
+## Setup for Using this Component
+
+### Priveleges
+
+Ensure that the pipeline has the appropriate RBAC priveleges to create the FybrikApplication from the pipeline.
+
+```bash
+kubectl apply -f rbac_resources.yaml -n kubeflow
+```
+
+### Storage for Write and Copy Flows
+
+[Register a storage account](https://fybrik.io/v1.0/samples/notebook-write/#deploy-resources-for-write-scenarios) in which the results can be written.  Example files are provided called kfp-storage-secret.yaml and kfp-storage-account.yaml.  Please change the values in these files with storage endpoint and credential details.  
+
+Note: Make sure that the endpoint is prefixed with https://
```

Comment:
`deploy-resources-for-write-scenarios` assumes that a user has created an account in object storage, probably the links should be both 
[Create an account in object storage](https://fybrik.io/v1.1/samples/notebook-write/#create-an-account-in-object-storage)
and
[Deploy resources for write scenarios ](https://fybrik.io/v1.1/samples/notebook-write/#deploy-resources-for-write-scenarios)

Reply:
The "Deploy resource for write scenario" contains 2 FybrikStorageAccounts, here we need only one. It can be confusing